SpatialConvolution.lua

```diff
@@ -318,9 +318,9 @@ function SpatialConvolution:createIODescriptors(input)
         maxBufSize = math.max(maxBufSize, bufSize[1])
 
         self.extraBuffer = self.extraBuffer or cudnn.getSharedWorkspace()
-        self.extraBufferSizeInBytes = self.extraBuffer:nElement() * 4 -- float
+        self.extraBufferSizeInBytes = self.extraBuffer:nElement() * cudnn.sizeof(self.weight)
```

Comment:
self.extraBuffer is still allocated as torch.CudaTensor
https://github.com/szagoruyko/cudnn.torch/blob/e9cd51dd0346d30894bd73a357677d9b494678b1/init.lua#L139
so to get its size in bytes you do need to multiply it by 4 and not by cudnn.sizeof(weight)